apps/back/src/Controller/ExpenseController.php

```diff
@@ -15,10 +16,26 @@
 #[Route('/expense', name: 'expense')]
 class ExpenseController extends AbstractController
 {
-    #[Route('/list', name: '_list')]
-    public function expensesList(): Response
+    #[Route('/list/{groupName}', name: '_list')]
+    public function expenseList(
+        EntityManagerInterface $em,
+        string $groupName,
+    ): Response
     {
-        return $this->render('expense/list.html.twig', []);
+        $groupRepository = $em->getRepository(Group::class);
+        $group = $groupRepository->findOneBy(['name' => $groupName]);
+
+        $expenseRepository = $em->getRepository(Expense::class);
+        $expenses = $expenseRepository->findBy(['group' => $group], ['date' => 'DESC']);
+
+        if($group !== null) {
+            return $this->render('expense/list.html.twig', [
+                'expenses' => $expenses,
+                'group' => $group,
```

Comment:
```suggestion
                'group' => $group,
```

il ne me semble pas nécessaire d'envoyer la liste des dépense du groupe car tu y as déjà accès depuis Twig avec `group.expenses`